OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -1320,12 +1319,14 @@ extension OmnipodPumpManager: PumpManager {
                     case .failure(let error):
                         self.log.default("Not recommending Loop because pump data is stale: %@", String(describing: error))
                         delegate?.pumpManager(self, didError: error)
+                        completion?()
```

Comment:
I think you need a completion call in the `case .none:` block as well.

Reply:
ah good catch!